### PR TITLE
chore: document image requirements, closes HYB-177

### PR DESCRIPTION
**Pull request by tpetillot**
**Motivation:**
Document image requirements

**Modification:**
Document requirements:
- AWS: jq, curl, java and cloud init
- Kubernetes: jq, curl and java
- Azure: no custom images for now

### CODE REVIEW

File: content/reference/admin/private_locations/configuration/ec2/index.md [hunk 1]
```diff
@@ -103,4 +100,13 @@ control-plane {
...
+### Custom AMI requirements
 
+AWS EC2 private locations AMI rely on some dependencies.
+
+So when using a custom AMI, make sure following are available:
+
+- [cloud-init](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html) integration.
+- [jq](https://jqlang.github.io/jq/download/) a lightweight and flexible command-line JSON processor.
+- [curl](https://curl.se/download.html) a command line tool and library for transferring data with URLs
+- [Java runtime environment](https://www.azul.com/downloads/?package=jre#zulu)
```
Comment (slandelle) [inline content/reference/admin/private_locations/configuration/ec2/index.md:112]: At least 8

**Review by slandelle — CHANGES_REQUESTED** [review]
What about Azure?

**Review by slandelle — CHANGES_REQUESTED** [review]
What about Azure?

**Comment by tpetillot** [comment]
@slandelle as specified in the commit message, we do not support custom images on Azure for now (which surprised me, because we did not go with batch job to support custom images cc @Obero)

**Review by slandelle — APPROVED** [review]

**Comment by tpetillot** [comment]
@slandelle feature now in back log for now ([HYB-236](https://gatlingcorp.atlassian.net/browse/HYB-236?atlOrigin=eyJpIjoiMGFhNWU5ZDI0OGU2NDVkYmE5ZDgwNTE3MjMzMDA4ZjIiLCJwIjoiaiJ9)).

Quoting @Obero : custom images for Azure have been put aside at the beginning, can become a priority if needed.

**Comment by slandelle** [comment]
@tpetillot OK, thanks! Not a priority atm as we're still to meet our first Azure Private Locations prospect.

**Review by slandelle — CHANGES_REQUESTED** [review]
Misses link to OS tuning: https://gatling.io/docs/gatling/reference/current/core/operations/

**Review by slandelle — CHANGES_REQUESTED** [review]
and that we only support OpenJDK https://gatling.io/docs/gatling/tutorials/installation/#java-version

**Review by bastien-gatling — APPROVED** [review]
Please not forget to squash your fixup ;) !

**Pull request merged**